src/components/Notification.vue

```diff
@@ -39,8 +39,8 @@ export default {
   max-width: rem(520px);
   margin-left: auto;
   margin-right: auto;
-  padding-top: env(safe-area-inset-top);
-  border-radius: 0 0 rem(8px) rem(8px);
+  margin-top: calc(env(safe-area-inset-top) + 10px);
```

Comment:
Please, use `rem` instead of raw pixels, and add a fallback for the case if `env()` is not supported.